modules/sdk-coin-sol/src/lib/transactionBuilder.ts

```diff
@@ -53,6 +53,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     this._transaction = tx;
     const txData = tx.toJson();
     this.sender(txData.feePayer as string);
```

Comment:
i wonder if this is going to break anything... i don't think so, but may be worth adding a check.

Reply:
updated with the sender